Дз №2

kat-khizhnayk opened the pull request.

kat-khizhnayk committed:
Дз №2

AlexFGeeR commented on js_homeworks/second.js:
```diff
@@ -0,0 +1,85 @@
+// 3 задача
+
+let num = +prompt('Введите число', '');
+let s = +prompt('Введите степень числа', '');
```
### Нейминг
По наименованию переменной непонятно, что в ней ожидать.

AlexFGeeR commented on js_homeworks/second.js:
```diff
@@ -0,0 +1,85 @@
+
+// 2 задача
+
+let n = +prompt('Введите число', '');
```
### Нейминг
По наименованию переменной непонятно, что в ней ожидать.

AlexFGeeR reviewed (commented):
Поправь нейминг

kat-khizhnayk committed:
Дз №3

- изменила нейминг в second.js
- сделала Дз №3 (third.js)

AlexFGeeR commented on js_homeworks/second.js:
```diff
@@ -0,0 +1,85 @@
+
+function isNumber (number) {
+    if (isNaN(number)) {
+        return "Ошибка";
```
### Лишний функционал.
Не обязательно указывать `return` какое-то значение, если ты его не планируешь использовать в будущем.

AlexFGeeR commented on js_homeworks/second.js:
```diff
@@ -0,0 +1,85 @@
+        return "Ошибка";
+    }
+
+for (let i = 0; i <= number; i++) {
+    if (i % 4 === 0) continue;
+    console.log(i);
+}
```
### Поехала немного вложенность.
Требуется добавить 1 таб на эти блоки.

AlexFGeeR commented on js_homeworks/second.js:
```diff
@@ -0,0 +1,85 @@
+
+    let result = 1;
+
+for (let i = 0; i<s; i++ ) {
+    result *= userNum;
+}
+return result;
```
### Вложенность.
Снова немного поехала вложенность и не видно к какому блоку относиться этот код.

AlexFGeeR commented on js_homeworks/second.js:
```diff
@@ -0,0 +1,85 @@
+    if (numb === rand) {
+        console.log(`Ура, вы победили! Ответ: ${rand}`);
+        break;
+        }
+    else {
```
### Отступы.
Во многих компаниях принятно писать данную конструкцию следующим образом -> `} else {`

kat-khizhnayk committed:
Все домашки до 9 лекции

Все домашки до 9 лекции:
- главный слайдер, несброс слайдера;
- слайдер swiper;
- несброс фильтра;
- пагинация;
- загрузка и отрисовка тегов и постов с сервера;
- прелоадер

kat-khizhnayk committed:
Готово!!!

kat-khizhnayk committed:
Изменения

kat-khizhnayk committed:
Update style.scss

kat-khizhnayk committed:
Окончательные изменения )
